subt/docker/robotika/run_solution.bash

```diff
@@ -28,13 +28,13 @@ done
 echo "Robot name is '$ROBOT_NAME'"
 
 echo "Starting ros<->osgar proxy"
-# Wait for ROS master, then start proxy
+# Wait for ROS master, then start ros nodes
 # http://wiki.ros.org/roslaunch/Commandline%20Tools#line-45
-roslaunch proxy sim.launch --wait robot_name:=$ROBOT_NAME &
+roslaunch robotika robot.launch --wait robot_name:=$ROBOT_NAME &
```

Comment:
And this is where things start colliding with the current state.

@zwn Do you see a way how to make it all externally configurable? Or maybe we should have two almost identical images, one for ground robots, one for drones? (I am assuming we would stick to at most one type of a ground robot and one type of a drone.)

Reply:
Yes. There is rosparam `/<robotName>/robot_description` that contains xml description of the robot. Its root element starts like this:
```
<robot name="ROBOTIKA_X2">
  <link name="base_link">
    <inertial>
...
```
so if anything, it contains the name of the robot configuration.

I believe I've already commented on this here https://github.com/robotika/osgar/pull/455#discussion_r413579034

Reply:
I changed that to make run_solution.bash usable for X2 and X4 without a change